boa/src/builtins/regexp/mod.rs

```diff
@@ -207,76 +252,190 @@ impl RegExp {
+    pub(crate) fn get_global(this: &Value, _: &[Value], context: &mut Context) -> Result<Value> {
+        Self::regexp_has_flag(this, 'g', context)
+    }
+
+    /// `RegExp.prototype.ignoreCase`
+    ///
+    /// The `ignoreCase` property indicates whether or not the "`i`" flag is used with the regular expression.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-get-regexp.prototype.ignorecase
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/ignoreCase
+    pub(crate) fn get_ignore_case(
+        this: &Value,
+        _: &[Value],
+        context: &mut Context,
+    ) -> Result<Value> {
+        Self::regexp_has_flag(this, 'i', context)
+    }
+
+    /// `RegExp.prototype.multiline`
+    ///
+    /// The multiline property indicates whether or not the "m" flag is used with the regular expression.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-get-regexp.prototype.multiline
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/multiline
+    pub(crate) fn get_multiline(this: &Value, _: &[Value], context: &mut Context) -> Result<Value> {
+        Self::regexp_has_flag(this, 'm', context)
+    }
+
+    /// `RegExp.prototype.dotAll`
+    ///
+    /// The `dotAll` property indicates whether or not the "`s`" flag is used with the regular expression.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-get-regexp.prototype.dotAll
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll
+    pub(crate) fn get_dot_all(this: &Value, _: &[Value], context: &mut Context) -> Result<Value> {
+        Self::regexp_has_flag(this, 's', context)
+    }
+
+    /// `RegExp.prototype.unicode`
+    ///
+    /// The unicode property indicates whether or not the "`u`" flag is used with a regular expression.
+    /// unicode is a read-only property of an individual regular expression instance.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-get-regexp.prototype.unicode
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/unicode
+    pub(crate) fn get_unicode(this: &Value, _: &[Value], context: &mut Context) -> Result<Value> {
+        Self::regexp_has_flag(this, 'u', context)
+    }
+
+    /// `RegExp.prototype.sticky`
+    ///
+    /// The `flags` property returns a string consisting of the [`flags`][flags] of the current regular expression object.
```

Comment:
This seems to be missing a link to the `flags`. It seems it's failing to resolve the link.